config/src/config/safety_rules_config.rs

```diff
@@ -123,15 +123,22 @@ impl ConfigSanitizer for SafetyRulesConfig {
 pub enum InitialSafetyRulesConfig {
     FromFile {
         identity_blob_path: PathBuf,
+        #[serde(skip_serializing_if = "Vec::is_empty", default)]
+        overriding_identity_paths: Vec<PathBuf>,
```

Comment:
Does this need to be a vector? Will 1 path not suffice (if the operator wants to rotate again, they need to move the new key to the old key location, and then add a single override). Would that work?

Reply:
In theory 2 slots (1 existing, 1 new) is enough. It's just i found n slots can also be supported for free by just replacing `Option<PathBuf>` with `Vec<PathBuf>`...